changelogs/fragments/4728-onepassword-v2.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
```

I think this is more than a bugfix. (Maybe even major changes?)
```suggestion
minor_changes:
```

---

It's a major under the hood change for more or less a bug fix. I'll go with a minor change.